search.py

```diff
@@ -336,42 +340,44 @@ def extend(U, open_dir, open_other, g_dir, g_other, closed_dir):
         open_dir.remove(n)
         closed_dir.append(n)
 
-        for c in problem.actions(n):
-            if c in open_dir or c in closed_dir:
-                if g_dir[c] <= problem.path_cost(g_dir[n], n, None, c):
+        for child in problem.actions(n):
+            if child in open_dir or child in closed_dir:
+                if g_dir[child] <= problem.path_cost(g_dir[n], n, None, child):
                     continue
 
-                open_dir.remove(c)
+                open_dir.remove(child)
 
-            g_dir[c] = problem.path_cost(g_dir[n], n, None, c)
-            open_dir.append(c)
+            g_dir[child] = problem.path_cost(g_dir[n], n, None, child)
+            open_dir.append(child)
 
-            if c in open_other:
-                U = min(U, g_dir[c] + g_other[c])
+            if child in open_other:
+                U = min(U, g_dir[child] + g_other[child])
 
         return U, open_dir, closed_dir, g_dir
 
     def find_min(open_dir, g):
         """Finds minimum priority, g and f values in open_dir"""
-        m, m_f = np.inf, np.inf
+        # pr_min_f isn't forward pr_min instead it's the f-value
+        # of node with priority pr_min.
+        pr_min, pr_min_f = np.inf, np.inf
```

Comment:
I agree, the new variable names are indeed more descriptive, nice catch!

Reply:
Thanks!